package.json

```diff
@@ -20,7 +20,18 @@
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-react": "^7.22.0",
+    "husky": "=4",
+    "lint-staged": ">=10",
```

Comment:
What's the advantage of using `>=` instead of a compatible range (`^`)?

Reply:
~I'm not actually not sure why either of these are set up this way.~ I generated this using `npx mrm lint-staged` as recommended [in the `prettier` docs](https://prettier.io/docs/en/precommit.html) to save time. This issue explains what's going on, at least for `husky`: https://github.com/sapegin/mrm/issues/145

I'll switch `lint-staged` to use compatible range.